packages/expo-screen-orientation/README.md

```diff
@@ -23,12 +23,31 @@ npm install expo-screen-orientation
 
 ### Configure for iOS
 
-Run `pod install` in the ios directory after installing the npm package.
+1. Run `pod install` in the ios directory after installing the npm package.
+2. Open the `AppDelegate.m` of your application.
+3. Import `<EXScreenOrientation/EXScreenOrientationViewController.h>`
+4. In `-application:didFinishLaunchingWithOptions:launchOptions` change default `root view controller` to `EXScreenOrientationViewController`:
+
+   Replace
+
+   ```objc
+   UIViewController *rootViewController = [UIViewController new];
+   ```
+
+   with:
+
+   ```objc
+   UIViewController *rootViewController = [[EXScreenOrientationViewController alloc] initWithDefaultScreenOrientationMask:UIInterfaceOrientationMaskPortrait]; // through parameter you can specify your default orientation mask.
```

Comment:
Can we default to using `init]`(less decisions for the developer) and note that people can also set a default orientation mask with a parameter?